src/main/java/bit/javaoop/Company.java

```diff
@@ -13,26 +13,26 @@ public Company(String name, CEO ceo) {
     private String name;
     private CEO ceo;
 
-    public void hireCEO(CEO c){
-        this.ceo=c;
+    public void hireCEO(CEO c) {
+        this.ceo = c;
     }
-    public String toString(){
-        if (ceo==null) return "NO CEO!";
-       StringBuilder sb = new StringBuilder();
-       sb.append(ceo.getName());
-       sb.append(" - CEO\n");
-       for (AbstractManager m:ceo.getManagers()){
-           sb.append("    ");
-           sb.append(m.getName());
-           sb.append(" - Manager\n");
-           for (AbstractEmployee e:m.getEmployees())
-           {
-               sb.append("        ");
-               sb.append(e.getName());
-               sb.append(" - Employee\n");
 
-           }
-       }
-    return sb.toString();
+    public String toString() {
+        if (ceo == null) return "NO CEO!";
+        StringBuilder sb = new StringBuilder();
+        sb.append(ceo.getName());
+        sb.append(" - CEO\n");
+        for (AbstractManager m : ceo.getManagers()) {
+            sb.append("\t");
+            sb.append(m.getName());
+            sb.append(" - Manager\n");
+            for (AbstractEmployee e : m.getEmployees()) {
+                sb.append("\t\t");
+                sb.append(e.getName());
+                sb.append(" - Employee\n");
```

Comment:
Builder to taki fajny pattern, w którym podczas implementowania metod robimy na końcu `return this;` czyli zwracamy samego siebie, pozwala to na takie fajne potokowe wywołania:
```
sb.append("\t\t")
    .append(e.getName())
    .append(" - Employee\n");
```